src/ChainRouter.php

```diff
@@ -370,4 +390,17 @@ public function hasRouters()
     {
         return 0 < count($this->routers);
     }
+
+    public function redirect(string $path, string $route, string $scheme = null): array
+    {
+        return [
+            '_controller' => 'Symfony\\Bundle\\FrameworkBundle\\Controller\\RedirectController::urlRedirectAction',
```

Comment:
we do not have a dependency on the framework-bundle in this component, so this won't always work.